config/HustleInc.yml

```diff
@@ -17,6 +17,8 @@ eng:
     - higgins
     - Lizard381
     - matthewferry
+    - MatrixFrog
+    - ibash
```

Comment:
This looks like it hasn't been updated in a while...
Nina is `ninachaubal`
Richard is `richardharrington`
Tim is `caplis`
Filip is `Filip-F`
King is `kingdavidmartins`
I don't think I've seen PRs from him, but we could add Bennett: `hustlerbb`

Reply:
Oh, also `robertbourbonnais`

Reply:
Is there no way to automatically pull this from the org so we don't have to do this manually every time?

Reply:
There probably is but I don't want to spend too much time on this. I will very easily fall into this trap if I'm not careful: https://xkcd.com/1629/

Reply:
I am also not on this list. We should probably add this to the onboarding instructions.